src/util/tubemap.js

```diff
@@ -841,31 +887,66 @@ function compareNoNodeReadsByPreviousY(a, b) {
   const segmentA = reads[a.readIndex].path[a.pathIndex];
   const segmentB = reads[b.readIndex].path[b.pathIndex];
   if (segmentA.order === segmentB.order) {
-    return a.previousY - b.previousY;
+    // we want to sort of the reverse when the segment is between a cycle
+    // this ensures a loop that starts on the outside, stays on the outside
+    // and loops are rolled in order
```

Comment:
This comment could be better written, I think. Maybe:

```suggestion
    // We want to sort in reverse order when the segment is along the reverse-going part of a cycle.
    // This ensures a loop that starts on the outside, stays on the outside,
    // and rolls up in order with other loops.
```